src/features/organizations/utils/generateTreeData.spec.ts

```diff
@@ -0,0 +1,207 @@
+import { generateTreeData } from './generateTreeData';
+import { TreeItemData } from '../rpc/getOrganizations';
+import { describe, expect, it } from '@jest/globals';
+import { ZetkinMembership, ZetkinOrganization } from 'utils/types/zetkin';
+
+interface mockOrganization {
+  id: number;
+  title: string;
+  parent?: mockOrganization;
+}
+
+interface mockMembership {
+  organization: {
+    id: number;
+    title: string;
+  };
+  role: string;
+}
```

Comment:
Why do you need these types? And why not use the function interfaces I suggested? 

```tsx
const organizations: ZetkinOrganization[] = [
  mockOrganization(1, 'Party A'),
  mockOrganization(2, 'Party B'),
]

const memberships: ZetkinMembership[] = [
  mockMembership(1, 'Party A', 'admin'),
  mockMembership(2, 'Party B', null),
]
```

Reply:
I thought that it will be more clear (and more helpful for me) if I create the object type that the mock functions are going to receive, other than that I follow the same pattern that you suggested. If you think the types are unnecessary, I'm okay removing them!